governance-exercises/Le Bateleur - Poll Exercise 01/Poll #2.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Increase Spork Protocol DAI Deposit Rate to 2.5% - July 24, 2023
+summary: Signal your support or opposition to increasing the Spork Protocol DAI Deposit Rate to 2.5%.
```

Comment:
If you're including the new number in the title. I might include the from x to y construction in the summary.